plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -761,6 +766,10 @@ public String toString() {
             }
             diskBuilder.append("/>\n");
 
+            if (_serial != null && !_serial.isEmpty() && _deviceType != DeviceType.LUN) {
```

Comment:
I see, but it is suddenly a new type of disk I haven't seen before in KVM. Hence my question.